app/cdap/components/PipelineTriggers/EnabledTriggersTab/EnabledInlineTriggerRow.tsx

```diff
@@ -114,75 +101,76 @@ const EnabledInlineTriggerRow = ({
-            <span>{T.translate(`${TRIGGER_PREFIX}.Events.KILLED`)}</span>
-          </CheckboxItemContainer>
-          <CheckboxItemContainer>
-            <Checkbox checked={failed} color="primary" size="small" />
-            <span>{T.translate(`${TRIGGER_PREFIX}.Events.FAILED`)}</span>
-          </CheckboxItemContainer>
-        </div>
-        {disableError && <ErrorText>{disableError}</ErrorText>}
-        <InlineTriggerActionButtonsContainer>
-          <PipelineTriggerButton
-            onClick={handlePayloadToggleClick}
-            data-cy={`${triggeringPipelineInfo.id}-view-payload-btn`}
-          >
-            {T.translate(`${PAYLOAD_PREFIX}.configPayloadBtnDisabled`)}
-          </PipelineTriggerButton>
+      <Card>
+        <CardContent>
+          <PipelineNameInDescription>
+            <strong>{T.translate(`${TRIGGER_PREFIX}.pipelineName`)}: </strong>
+            <PipelineName>{trigger.programId.application}</PipelineName>
+          </PipelineNameInDescription>
+          <div>
+            <strong>{T.translate(`${TRIGGER_PREFIX}.namespace`)}: </strong>
+            <StyledNameSpace>{trigger.programId.namespace}</StyledNameSpace>
+          </div>
+          <div>
+            <strong>{T.translate(`${TRIGGER_PREFIX}.description`)}: </strong>
+            <span>{expandedTriggerInfo && expandedTriggerInfo.description}</span>
+          </div>
+          <HelperText>{T.translate(`${TRIGGER_PREFIX}.helperText`, { pipelineName })}</HelperText>
+          <CheckboxContainer>
+            <CheckboxItemContainer>
+              <Checkbox checked={completed} color="primary" size="small" />
+              <span>{T.translate(`${TRIGGER_PREFIX}.Events.COMPLETED`)}</span>
+            </CheckboxItemContainer>
+            <CheckboxItemContainer>
+              <Checkbox checked={killed} color="primary" size="small" />
+              <span>{T.translate(`${TRIGGER_PREFIX}.Events.KILLED`)}</span>
+            </CheckboxItemContainer>
+            <CheckboxItemContainer>
+              <Checkbox checked={failed} color="primary" size="small" />
+              <span>{T.translate(`${TRIGGER_PREFIX}.Events.FAILED`)}</span>
+            </CheckboxItemContainer>
+          </CheckboxContainer>
+          {disableError && <ErrorText>{disableError}</ErrorText>}
           <PayloadConfigModal
             isOpen={payloadModalOpen}
             triggeringPipelineInfo={triggeringPipelineInfo}
             triggeredPipelineInfo={triggeredPipelineInfo}
             onToggle={handlePayloadToggleClick}
-            scheduleInfo={groupSchedule}
+            scheduleInfo={compositeTriggerSchedule}
             disabled={true}
             pipelineCompositeTriggersEnabled={true}
           />
-        </InlineTriggerActionButtonsContainer>
-      </div>
+        </CardContent>
+        <CardActions>
+          <TriggerCardButton
+            // The Anchor tab is not working, using this hacky way to fix it for now
```

Comment:
Again here - please describe to me what happens when this anchor tag is not working?

Reply:
Yeah the page is not refreshing but showed the re-direct url on the bottom